Translations update from Weblate

weblate-fpf opened the pull request:
Translations update from [Weblate](https://weblate.securedrop.org) for [SecureDrop/SecureDrop](https://weblate.securedrop.org/projects/securedrop/securedrop/).


It also includes following components:

* [SecureDrop/desktop](https://weblate.securedrop.org/projects/securedrop/desktop/)



Current translation status:

![Weblate translation status](https://weblate.securedrop.org/widget/securedrop/securedrop/horizontal-auto.svg)

eloquence commented:
Please do not merge just yet; going to test updates to the branch from Weblate with the current branch protection rules in effect.

purplepenguin73 commented:
Are there plans to add the other languages with significant progress? (Arabic, Czech, Norwegian, Persian, Polish, etc.)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: Meng Pang Wang <mengpangwang@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/zh_Hant/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Translated using Weblate (Persian)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Maryam Azad <hannahperez8@protonmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/fa/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (German)

Currently translated at 100.0% (357 of 357 strings)

Translated using Weblate (German)

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: Erik Moeller <eloquence@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/de/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Catalan)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: John Smith <ecron_89@hotmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/ca/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Translated using Weblate (Esperanto)

Currently translated at 88.8% (8 of 9 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 100.0% (357 of 357 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 100.0% (357 of 357 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Merge remote-tracking branch 'origin/develop' into develop

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Translated using Weblate (Esperanto)

Currently translated at 90.0% (9 of 10 strings)

Co-authored-by: Pseudolocale add-on <noreply-addon-pseudolocale@weblate.org>
Co-authored-by: Weblate <noreply@weblate.org>
Co-authored-by: anonymous <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/eo/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/eo/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Croatian)

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/hr/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Italian)

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: Claudio Arseni <claudio.arseni@ubuntu.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/it/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (French)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: AO Localization Lab <ao@localizationlab.org>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/fr/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Icelandic)

Currently translated at 100.0% (356 of 356 strings)

Translated using Weblate (Icelandic)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Icelandic)

Currently translated at 100.0% (357 of 357 strings)

Co-authored-by: Oktavia <oktaviahrund@gmail.com>
Co-authored-by: Sveinn í Felli <sv1@fellsnet.is>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/is/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: SecureDrop/SecureDrop
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translation: SecureDrop/SecureDrop

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translated using Weblate (Polish)

Currently translated at 100.0% (356 of 356 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Adam Rak <kitynska@gmail.com>
Co-authored-by: Weblate <noreply@weblate.org>
Translate-URL: https://weblate.securedrop.org/projects/securedrop/desktop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/
Translate-URL: https://weblate.securedrop.org/projects/securedrop/securedrop/pl/
Translation: SecureDrop/SecureDrop
Translation: SecureDrop/desktop
